examples/create-react-app/package.json

```diff
@@ -5,11 +5,13 @@
   "scripts": {
```

Comment:
perhaps revert this (and the one in ssr)? issue with the react-scripts package hoist to the root node_modules dir

Reply:
done